README.md

```diff
@@ -5,18 +5,19 @@ cloudformation-builder
 [![codecov](https://codecov.io/gh/StuPro-TOSCAna/cloudformation-builder/branch/master/graph/badge.svg)](https://codecov.io/gh/StuPro-TOSCAna/cloudformation-builder)
 
 
-CloudFormation-Builder is a Java 8 DSL for creating AWS CloudFormation templates.
+CloudFormation-Builder is a project in Java 8 for creating AWS CloudFormation templates.
```

Comment:
I kind of liked using domain-specific language (DSL) to describe the cloudformation-builder, so I'd be in favour of reverting this.